src/legacy/core_plugins/vega/public/index.ts

```diff
@@ -17,20 +17,7 @@
  * under the License.
  */
 
+import { VegaPlugin } from './plugin';
+import { plugins } from './setup';
 
-import 'ngreact';
-
-import { wrapInI18nContext } from 'ui/i18n';
-import { uiModules } from 'ui/modules';
-const module = uiModules.get('kibana/vega', ['react']);
-
-import { VegaHelpMenu } from './vega_help_menu';
-import { VegaActionsMenu } from './vega_action_menu';
-
-module.directive('vegaActionsMenu', function (reactDirective) {
-  return reactDirective(wrapInI18nContext(VegaActionsMenu));
-});
-
-module.directive('vegaHelpMenu', function (reactDirective) {
-  return reactDirective(wrapInI18nContext(VegaHelpMenu));
-});
+export const plugin = new VegaPlugin().setup(plugins);
```

Comment:
In `index.ts` file there should be a plugin initializer instead: https://github.com/streamich/kibana/blob/shim-embeddable/src/legacy/core_plugins/embeddable_api/public/shim/public/index.ts